### PR TITLE
Added new Rune sequence proc

### PR DESCRIPTION
**Pull request by apense**
In reference to #2353, but didn't use the name `runes` (just to avoid confusion with the iterator). Can change whatever. Also put documentation into imperative, capitalized form. Also added some missing periods. I think that's it.
To the trailing periods: do these go there or not? Maybe we could do: put periods if there is more than one sentence.

### CODE REVIEW

File: lib/pure/unicode.nim [hunk 1]
```diff
@@ -1163,7 +1163,7 @@ proc binarySearch(c: RuneImpl, tab: openArray[RuneImpl], len, stride: int): int
   return -1
 
 proc toLower*(c: Rune): Rune {.rtl, extern: "nuc$1", procvar.} =
-  ## Converts `c` into lower case. This works for any Unicode character.
+  ## Convert `c` into lower case. This works for any Unicode character.
```
Comment (dom96) [inline lib/pure/unicode.nim]: Not sure whether this is a valid change. "Converts" sounds better to me.

File: lib/pure/unicode.nim [hunk 2]
```diff
@@ -1259,8 +1260,14 @@ iterator runes*(s: string): Rune =
     fastRuneAt(s, i, result, true)
     yield result
 
+proc runesIn*(s: string): seq[Rune] =
```
Comment (Araq) [inline lib/pure/unicode.nim]: Not acceptable name. Suggestion: ``toRunes``.

**Comment by apense** [comment]
changed to `toRunes`

**Comment by Perelandric** [comment]
@apense:  Just curious, why the changes to the docs where `returns/converts/etc` is now `return/convert/etc`?

IMO, it reads more naturally with the `s` as a description of what it does. Without, it sounds like it's meant to be an instruction of what the developer should do, like in a callback.

**Comment by apense** [comment]
@Perelandric It came from [here](https://github.com/nim-lang/Nim/pull/3035). There was a little discussion. It will show up in the documentation guidelines because of that pull. Whenever I make a PR, I usually just go ahead and change it.

**Comment by Araq** [comment]
@Perelandric I agree with you but if we copy from Python the uppercase we should also copy the stupid imperative form.

**Comment by Varriount** [comment]
@araq why? Surely we can pick and choose.

**Comment by apense** [comment]
I made this PR before I made #3089. I can change over to the "official
style" when I get to a computer (I'm emailing right now).
I haven't made many of those types of changes, luckily.
The style guide PR is the place to figure all this out before moving on
here.

**Comment by Araq** [comment]
Ok, so Uppercase plus the 's'. And now let's please stop discussing this.

**Comment by apense** [comment]
Great. should be ready

File: lib/pure/unicode.nim [hunk 3]
```diff
@@ -1322,3 +1330,4 @@ when isMainModule:
   assert reversed("先秦兩漢") == "漢兩秦先"
   assert reversed("as⃝df̅") == "f̅ds⃝a"
   assert reversed("a⃞b⃞c⃞") == "c⃞b⃞a⃞"
+  assert len(runesIn("as⃝df̅")) == runeLen("as⃝df̅")
```
Comment (Araq) [inline lib/pure/unicode.nim]: 'runesIn' has been renamed to 'toRunes' :P

**Comment by apense** [comment]
Should actually be ready now

**Pull request merged**